de.schmidhuberj.tubefeeder.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "de.schmidhuberj.tubefeeder",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": { "org.freedesktop.Platform.ffmpeg-full": {
+        "directory": "lib/ffmpeg",
+        "version": "21.08beta",
+        "add-ld-path": "."
+      }
+    },
+    "command": "tubefeeder",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+	"--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+	"--filesystem=xdg-data/tubefeeder:create",
+	"--filesystem=xdg-cache/tubefeeder:create",
+	"--filesystem=xdg-config/mpv:ro"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "test-args": [
+            "--socket=x11"
```

Comment:
there is no X11 on the buidlers either.

Reply:
I think test-args are skipped in sandboxed mode anyway.